core/src/main/java/io/grpc/internal/BinaryLogProvider.java

```diff
@@ -280,8 +214,16 @@ public CallId(long hi, long lo) {
       this.lo = lo;
     }
 
-    static CallId fromCensusSpan(Span span) {
+    public static CallId fromCensusSpan(Span span) {
       return new CallId(0, ByteBuffer.wrap(span.getContext().getSpanId().getBytes()).getLong());
     }
   }
+
+  public CallId getServerCallId() {
```

Comment:
Where are these methods called?

Reply:
I was probably overzealous in making this PR focused on only changing the generation. I plugged it into BinaryLogProviderImpl now.